client-library/src/index.ts

```diff
@@ -1 +1 @@
-export { createHonoMiddleware } from "./honoMiddleware";
+export { createHonoMiddleware } from "./honoMiddleware.js";
```

Comment:
note - these changes with the `.js` extension are because this is branched off of https://github.com/fiberplane/fpx/pull/79